meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio_%.bbappend

```diff
@@ -18,6 +18,8 @@ do_deploy_append_mx8m() {
                     else
                         install -m 0644 ${B}/${config}/arch/arm/dts/${UBOOT_DTB_NAME} ${DEPLOYDIR}/${BOOT_TOOLS}
                     fi
+                    install -m 0644 ${B}/${config}/dts/dt-spl.dtb ${DEPLOYDIR}/${BOOT_TOOLS}
+                    install -m 0644 ${B}/${config}/spl/u-boot-spl-nodtb.bin ${DEPLOYDIR}/${BOOT_TOOLS}
```

Comment:
This binary is specific to the config, so I believe it is better to follow the same pattern as used by u-boot-nodtb.bin (e.g.`u-boot-spl-nodtb.bin-${MACHINE}-${UBOOT_CONFIG}`).

This is usually done by OE as a way to support building multiple configs in one shot.